src/billing/ConsolidatedSpendReport.tsx

```diff
@@ -0,0 +1,427 @@
+import { Link } from '@terra-ui-packages/components';
+import { subDays } from 'date-fns/fp';
+import _ from 'lodash/fp';
+import React, { ReactNode, useEffect, useState } from 'react';
+import { DateRangeFilter } from 'src/billing/Filter/DateRangeFilter';
+import { SearchFilter } from 'src/billing/Filter/SearchFilter';
+import { BillingAccountStatus, parseCurrencyIfNeeded } from 'src/billing/utils';
+import { BillingProject } from 'src/billing-core/models';
+import { fixedSpinnerOverlay } from 'src/components/common';
+import { ariaSort, HeaderRenderer } from 'src/components/table';
+import { Ajax } from 'src/libs/ajax';
+import { Billing } from 'src/libs/ajax/billing/Billing';
+import {
+  AggregatedWorkspaceSpendData,
+  SpendReport as SpendReportServerResponse,
+} from 'src/libs/ajax/billing/billing-models';
+import { Metrics } from 'src/libs/ajax/Metrics';
+import Events, { extractBillingDetails } from 'src/libs/events';
+import * as Nav from 'src/libs/nav';
+import { memoWithName, useCancellation } from 'src/libs/react-utils';
+import { SpendReportStore, spendReportStore } from 'src/libs/state';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+import { GoogleWorkspace, GoogleWorkspaceInfo, WorkspaceWrapper } from 'src/workspaces/utils';
+
+// Copied and slightly altered from WorkspaceCard and WorkspaceCardHeaders in Workspaces,
+// May want to instead extend them
+const workspaceLastModifiedWidth = 150;
+
+interface ConsolidatedSpendWorkspaceCardHeadersProps {
+  sort: { field: string; direction: 'asc' | 'desc' };
+  onSort: (sort: { field: string; direction: 'asc' | 'desc' }) => void;
+}
+
+// Since the variable names don't match display names for these two columns, we need to map them
+const columnTitleToVariableNameMap = {
+  billingAccount: 'namespace',
```

Comment:
This should map to `billingProject` here and throughout the consolidated report. We tend to reserve "account" for when we're talking about the Google billing account

Reply:
Meaning just that the words the user should see should be 'Billing Project' instead of 'Billing Account'?